addon/components/frost-link.js

```diff
@@ -83,19 +71,25 @@ export default LinkComponent.extend(PropTypeMixin, {
       design: '',
       icon: '',
       priority: '',
-      size: '',
-      text: ''
```

Comment:
In order to data-drive links I'd like to see if we can figure out how to keep `text` (and make it actually work) - I think we can piggyback on `linkTitle` for this https://github.com/emberjs/ember.js/blob/v2.7.0/packages/ember-htmlbars/lib/components/link-to.js#L796